usb/device/targets/TARGET_Cypress/TARGET_PSOC6/USBPhy_PSOC6.cpp

```diff
@@ -0,0 +1,378 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates.
```

Comment:
this PR still is adding additional functionaity (USB here) ? It should be also separate PR

Reply:
OK, removed commit that adds USB device implementation.

Is it appropriate to create pull request with USB device functionality BEFORE this PR is merged to mainline? Note, the implementation depends on psoc6hal and psoc6pdl additions introduced by this PR.